cli/dcoscli/config/main.py

```diff
@@ -109,6 +109,38 @@ def _unset(name):
 
     return 0
 
+def _format_config(file_value, effective_value, name=None, envvar_name=None,
+                   left_pad_fmt=None):
```

Comment:
Alternatively, the caller could simply tell the formatter whether it's an override case, which would enable password formatting to be handled in this function too.